### ISSUE TITLE
[WCAG] Ensure keyboard navigation - Legend Panel component

### ISSUE THREAD

**Issue by kenchase** [Canadian-Geospatial-Platform/geoview#3213]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Current Behavior

Layer items are not accessible via keyboard (tab key) within the legend panel.

### Expected Behavior

Layer items ned to be accessible via keyboard (tab key) within the legend panel
